config/config.yaml.dist

```diff
@@ -9,3 +9,4 @@ Location: [0,0]
 PrimeURL: "ws://127.0.0.1:8547"
 RegionURLs: ["ws://127.0.0.1:8579", "ws://127.0.0.1:8581", "ws://127.0.0.1:8583"]
 ZoneURLs: [["ws://127.0.0.1:8611", "ws://127.0.0.1:8643", "ws://127.0.0.1:8675"], ["ws://127.0.0.1:8613", "ws://127.0.0.1:8645", "ws://127.0.0.1:8677"], ["ws://127.0.0.1:8615", "ws://127.0.0.1:8647", "ws://127.0.0.1:8679"] ]
+RunBlake3: False
```

Comment:
Personally I feel hesitant about including this flag, just because it's going to be totally deprecated. But we also distribute it in go-quai.
@wizeguyy what do you think?

Reply:
it sucks switching between branches constantly

I could use worktrees I guess but couldn't we just add an issue to remove later?

Reply:
I was only referring to having the flag in the dist or not. I agree this is a much needed change.

These are the three scenarios:
- Copy the dist into network.env and run: runs progpow
- Add runblake3: false: runs progpow
- Add runblake3: true: runs blake3

Reply:
Discussed offline